src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,108 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
+    private RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
```

Comment:
Consider using `@BeforeEach` to initialize the `RegistrationServiceImpl` instance if its state should reset before each test. If the instance doesn't hold state and can be shared, use `@BeforeAll` with a static initialization.